DNN Platform/DotNetNuke.ModulePipeline/ModuleControlPipeline.cs

```diff
@@ -0,0 +1,194 @@
+﻿using System;
+using System.IO;
+using DotNetNuke.Entities.Modules;
+using DotNetNuke.Instrumentation;
+using DotNetNuke.Services.Localization;
+using DotNetNuke.UI.Modules.Html5;
+using DotNetNuke.UI.Modules;
+using DotNetNuke.Web.Razor;
+using System.Collections.Generic;
+using DotNetNuke.Web.Mvc;
+
+#if NET472
+using System.Web.UI;
+#endif
+
+namespace DotNetNuke.ModulePipeline
+{
+    /// <summary>
+    /// The Module Pipeline that determines which Module pattern
+    /// to invoke based on the input module type.
+    /// </summary>
+    public class ModuleControlPipeline
+        // MULTI-TARGETTING PIPELINE
+        // -------------------------
+        // This file multi-targets .NET Framework and .NET Standard,
+        // which is needed as DNN migrates to .NET Core. The 'NET472'
+        // pre-processor directives are to fully support Legacy DNN.
+        // As the Pipeline is upgraded to be more complaint with 
+        // .NET Standard 2.0 use the apprioprate pre-processor directives.
+#if NET472
+        : IModuleControlPipeline
+#endif
+    {
+        private static readonly ILog TracelLogger = LoggerSource.Instance.GetLogger("DNN.Trace");
```

Comment:
Is this a typo?  Should it be `TraceLogger` instead of `TracelLogger`?

Reply:
(I understand this was copied from other code, so the original (lack of) intention may be lost)

Reply:
I just copied and pasted this code it needs to be updated to `TraceLogger` thanks for pointing this out.